src/components/Button/index.ts

```diff
@@ -1,52 +1,75 @@
-import Element from '../Element';
+import Element from '../Element/index';
 
 const CLASS_BUTTON = 'pcui-button';
 
+namespace Button {
+    export interface Args extends Element.Args {
+        /**
+         * If unsafe is true, text will be set on the dom's innerHTML rather than textContent.
+         */
+        unsafe?: boolean;
+        /**
+         * Sets the text of the button.
+         */
+        text?: string,
+        /**
+         * The CSS code for an icon for the button. e.g. E401 (notice we omit the '\' character).
```

Comment:
Just wondering....are these codes actually documented anywhere?

Reply:
Not officially no. This issue is tracked here: https://github.com/playcanvas/pcui/issues/183